docs/submission/submitting-molecular-data.md

```diff
@@ -12,161 +15,688 @@ Raw molecular data is submitted to a **Regional Data Processing Centre (RDPC)**.
+
+- Sanity Checks (`sanityCheck`)
+- [Optional] Download data from EGA Archives (`EgaDownloadWf`)
+- [Optional] Cram to Bam conversion (`cram2bam`)
+- Generate Metadata Payload (`pGenExp`)
+- Validate Molecular Data and Metadata (`valSeq`)
+- Upload Files to RDPC (`uploadWf`)
+- Generate submission receipt (`submissionReceipt`)
+
+### Quick Start
 
-tar xvzf song-client.tar.gz
+1. Install [Nextflow](https://www.nextflow.io/) (>=22.10.0)
+2. Install [Docker](https://docs.docker.com/engine/installation/) or [Singularity](https://www.sylabs.io/guides/3.0/user-guide/) for full workflow reproducibility
+3. Download the workflow and do a dry run submission with the following commands
 
-## Note: Once unzipped, the final directory will be suffixed with the latest release number.
-cd song-client-<latest-release-number>
 ```
+git clone https://github.com/icgc-argo/argo-data-submission.git
+cd argo-data-submission
+nextflow run -preview main.nf -params-file example-params.local.json
+```
+
+This will give you a list of processes that will be invoked with the given input params for submitting data from local storage. E.g,
+
+```
+N E X T F L O W  ~  version 22.10.0
+* PREVIEW * null [fabulous_allen] DSL2 - revision: 5d3c543462
+[-        ] process > ArgoDataSubmissionWf:sanityCheck        -
+[-        ] process > ArgoDataSubmissionWf:checkCramReference -
+[-        ] process > ArgoDataSubmissionWf:pGenExp            -
+[-        ] process > ArgoDataSubmissionWf:valSeq             -
+[-        ] process > ArgoDataSubmissionWf:uploadWf:songSub   -
+[-        ] process > ArgoDataSubmissionWf:uploadWf:songMan   -
+[-        ] process > ArgoDataSubmissionWf:uploadWf:scoreUp   -
+[-        ] process > ArgoDataSubmissionWf:uploadWf:songPub   -
+[-        ] process > ArgoDataSubmissionWf:submissionReceipt   -
+```
+
+Please verify if you’ve provided the required params to invoke the expected processes.
+
+4. Start submitting your own data following the steps in section [How to Upload Molecular Data](/docs/submission/submitting-molecular-data#how-to-upload-molecular-data)
+
+## How to Upload Molecular Data
+
+Molecular data files are submitted in conjunction with necessary, descriptive data about molecular files and samples, called metadata. The DCC requires that metadata is submitted at the same time as the molecular data to facilitate automated downstream analysis. This metadata is requested to be one of the mandatory inputs for the data submission workflow. Before proceeding, please read the instructions on how to prepare and format the metadata according to [molecular metadata model](/docs/submission/submitting-molecular-data#understanding-the-metadata-model) and [validation rules](/docs/submission/submitting-molecular-data#molecular-data-and-metadata-validation-rules).
```

Comment:
Molecular data files are submitted in conjunction with necessary, descriptive data about molecular files and samples, called metadata. The DCC requires metadata to be submitted at the same time as the molecular data to facilitate automated downstream analysis. The metadata is one of the mandatory inputs for the data submission workflow. Before proceeding, please read the instructions on how to prepare and format the metadata according to [molecular metadata model](/docs/submission/submitting-molecular-data#understanding-the-metadata-model) and [validation rules](/docs/submission/submitting-molecular-data#molecular-data-and-metadata-validation-rules).